Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -858,4 +859,17 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="compression_lzma2">LZMA2 (slow)</string>
     <string name="compression_zstandard">Zstandard (recommended)</string>
 
+    <!-- About dialog -->
+    <string name="about_dolphin">Dolphin</string>
+    <string name="wark" translatable="false">WARK WARK WARK</string>
+    <string name="about_dolphin_image">Dolphin logo</string>
+    <string name="about_branch">Branch:\n%s</string>
+    <string name="about_revision">Revision:\n%s</string>
+    <string name="about_check_for_updates">Check for updates: dolphin-emu.org/download</string>
+    <string name="about_dolphin_description">Dolphin is a free and open-source GameCube and Wii emulator.\nThis software should not be used to play games you do not legally own.</string>
```

Comment:
Having exactly one line break looks a little odd IMO. What do you think about having two?

Reply:
Just tried it and it looks good.